src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -8681,6 +8697,25 @@ public void PathMapParser()
             Assert.Equal((int)ErrorCode.ERR_InvalidPathMap, parsedArgs.Errors[0].Code);
         }
 
+        [WorkItem(7588, "https://github.com/dotnet/roslyn/issues/7588")]
+        [Fact]
+        public void Version()
+        {
+            var expected = FileVersionInfo.GetVersionInfo(typeof(CSharpCompiler).Assembly.Location).FileVersion;
+
+            var outWriter = new StringWriter(CultureInfo.InvariantCulture);
+            var csc = new MockCSharpCompiler(null, _baseDirectory, new[] { "/version" });
```

Comment:
>new MockCSharpCompiler [](start = 22, length = 22)

I think we should actually run real command line compiler, there are many tests like this in this file, see tests that use ProcessUtilities.RunAndGetOutput helper.